content/v3/serverless-ai-api-guide.md

```diff
@@ -247,7 +247,7 @@ func init() {
 
 ## Troubleshooting
 
-### Error "Local LLM operations are not supported in this version of Spin"
+### Error "Local LLM Operations Are Not Supported in This Version of Spin"
```

Comment:
No, this should preserve the case of the message

Reply:
> Are these changes implying that the linter will block a PR for having two blank lines in succession? Was it there before? Is that really worthwhile? It all seems a bit aggressive.

Right, this was _also_ an error, on top of the blank lines!  I admit I don't know why these are cropping up now since we didn't see them in fermyon/developer.  I'll dig in and try to find the reason for its strictness...

Reply:
Holy cow, now the linter on the _old_ site has run berserk.  Absolutely unusable.  I wonder what's happened.

Reply:
Okay the linter on the old site was happy in CI a few weeks back, is happy locally, and yet is berserk in CI now with no changes.  I am not sure what is going on.  If the quickest fix for this repo is to turn the linter off, then turn the linter off, because something is mucked up.

Reply:
Thanks for investigating.  Created a PR to defer linting for now.